### PR TITLE
Update .env.example

### PR DESCRIPTION
**Pull request by hey-hoho**
The parameter format remains consistent with the document.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -17,4 +17,4 @@ TIDB_ENABLE_SSL=false
 # DATABASE_URL=mysql://root@localhost:4000/test
 
 # Example 2: Connect to TiDB Serverless with public endpoint (Requried TLS connection)
-# DATABASE_URL=mysql://{talent_id}.root:{password}@gateway01.{region_id}.prod.aws.tidbcloud.com:4000/{database}?ssl={"minVersion":"TLSv1.2"}
\ No newline at end of file
+# DATABASE_URL=mysql://{talent_id}.root:{password}@gateway01.{region_id}.prod.aws.tidbcloud.com:4000/{database}?ssl={"minVersion":"TLSv1.2"}
```
Comment (Mini256) [inline .env.example]:
```suggestion
# DATABASE_URL='mysql://{talent_id}.root:{password}@gateway01.{region_id}.prod.aws.tidbcloud.com:4000/{database}?ssl={"minVersion":"TLSv1.2"}'
```

File: .env.example [hunk 2]
```diff
@@ -17,4 +17,4 @@ TIDB_ENABLE_SSL=false
 # DATABASE_URL=mysql://root@localhost:4000/test
```
Comment (Mini256) [inline .env.example]:
```suggestion
# DATABASE_URL='mysql://root@localhost:4000/test'
```

File: .env.example [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
 # Weather enable TLS (SSL) connection. (**MUST** set up to `true` for connecting TiDB Serverless Cluster via Public Endpoint)
-TIDB_ENABLE_SSL=false
+TIDB_ENABLE_SSL='false'
 
 # Provide CA certificate to overwrite the default CA certificate. (Optional for TiDB Serverless Cluster)
-# TIDB_CA_PATH=/path/to/ca.pem
+# TIDB_CA_PATH='/path/to/ca.pem'
 
 # (Option 2) Connect to TiDB cluster with Connection URL
 # DATABASE_URL=mysql://{user}:{password}@{host}:{port}/{database}
```
Comment (Mini256) [inline .env.example]:
```suggestion
# DATABASE_URL='mysql://{user}:{password}@{host}:{port}/{database}'
```

File: .env.example (hunk 1 above)
Comment (hey-hoho) [reply .env.example]: get

**Review by Mini256 — APPROVED** [review]
LGTM, thanks a lot!

**Pull request merged**